terraform/projects/app-ecs-albs/main.tf

```diff
@@ -128,7 +128,7 @@ resource "aws_acm_certificate" "monitoring_cert" {
 
 resource "aws_route53_record" "monitoring_cert_validation" {
   # Count matches the domain_name plus each `subject_alternative_domain`
-  count = "${1 + length(concat(aws_route53_record.prom_alias.*.fqdn, aws_route53_record.alerts_alias.*.fqdn))}"
+  count = "${1 + (2 * length(data.terraform_remote_state.infra_networking.public_subnets))}"
```

Comment:
This change should not be in this commit it's unrelated to the commit and there is another pull request which fixes the problem.

Reply:
We understand but without this change the PR will not function as expected. The alternative PR didn't merge so we just took some of the logic.

Reply:
I am not sure what you would like to do about this but it is up to you to fair.

Reply:
I've updated the PR which should resolve this issue - https://github.com/alphagov/prometheus-aws-configuration-beta/pull/47, so that just needs to be approved and merged in, then you can rebase off master.